spec/integration/user_show_page

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'User Show Page', type: :feature do
+  let!(:user) { create(:user_with_comments_and_likes) }
+
+  it 'displays user information correctly' do
+    visit user_path(user)
+
+    expect(page).to have_content(user.username)
+    expect(page).to have_selector("img[src$='#{user.profile_picture_url}']")
+    expect(page).to have_content("Posts: #{user.posts.count}")
+    expect(page).to have_content(user.bio)
+
+    user.posts.first(3).each do |post|
+      expect(page).to have_content(post.title)
+    end
+
+    click_link 'View All Posts'
+    expect(current_path).to eq(user_posts_path(user))
+  end
+end
```

Comment:
- [ ] Great work writing and making all your tests for this file passed. You've done a great work guys. However, according to the project requirements. There're couple of tests that are yet to be written. KIndly look at those underline points. Tests are yet to be written for them.

### Passed tests
![test-passed](https://github.com/Ashnaali3255/Blog-website/assets/26597432/8235735e-d394-4a48-86d4-3055fb71a2dc)


![user-show-page](https://github.com/Ashnaali3255/Blog-website/assets/26597432/a57f4d2d-c3e4-413d-bd82-818642519074)

Kindly attend to them too. :+1: